spring/boot3-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaAutoConfiguration.java

```diff
@@ -69,16 +69,16 @@ public Server armeriaServer(
             ArmeriaSettings armeriaSettings,
             InternalServices internalService,
             Optional<MeterRegistry> meterRegistry,
-            Optional<List<MetricCollectingServiceConfigurator>> metricCollectingServiceConfigurators,
+            ObjectProvider<MetricCollectingServiceConfigurator> metricCollectingServiceConfigurators,
```

Comment:
Note: This API has been around since 4.3, so there's probably next to no user impact. The change is now that beans are correctly ordered by spring's priority system